Common/UnitDefinitions/Mass.json

```diff
@@ -210,6 +210,40 @@
           "Abbreviations": [ "gr" ]
         }
       ]
+    },
+    {
+      "SingularName": "SolarMass",
+      "PluralName": "SolarMasses",
+      "BaseUnits": {
```

Comment:
You can remove `BaseUnits` (plural). This is a new experimental concept we are playing with and the details are not all sorted out yet.

Reply:
Done. It nows compile properly.